### PR TITLE
Add extension to query upcoming holidays

### PR DESCRIPTION
**Pull request by rodrigouroz**
## Description

This adds an extension to query the upcoming holidays for a country. It uses in the background the site https://askholidays.vercel.app

## Screencast

https://user-images.githubusercontent.com/384037/171435315-0f30e305-c4ff-4a38-8134-0472fa2958ba.mov

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/holidays/package.json [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  "categories": [
+    "Data",
+    "Productivity"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "holidays",
+      "title": "Next Holidays",
+      "subtitle": "upcoming holidays for a country",
```
Comment (pernielsentikaer) [inline extensions/holidays/package.json:17]:
```suggestion
      "title": "List Upcoming Holidays",
```

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi @rodrigouroz 👋 

Thanks for your contribution, it's a very nice one 🔥 

What about calling the extension and folder for "Upcoming Holidays" instead, next sounds wrong in my ears 😄 

Would it be possible to include something like [SWR](https://swr.vercel.app) to cache the data, it feels very heavy when browsing the list and revisiting the same elements.

An idea for this or next release could be a possibility to pin countries 🔥 

I'm looking forward to testing this extension again 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

**Comment by rodrigouroz** [comment]
Done. I loved the idea of pinning countries, I'll do it in another PR not to hold on this first version.

**Comment by pernielsentikaer** [comment]
Hi @rodrigouroz 👋 

Awesome update, like the new features! 🔥 

I added new metadata images; the current was in the wrong format 😄 

One thing that would also be cool to add is a ´cache adapter` to cache the results for a long time since they don't nearly ever change 😄 

You can see a bit more about how it's done in the [Sentry extension](https://github.com/raycast/extensions/blob/df23810f3c604686d55c51f13add6cba9c445285/extensions/sentry/src/cache.ts) - Do you think you could look into this?

I'm looking forward to testing this extension again 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

**Review by pernielsentikaer — APPROVED** [review]
Awesome, @rodrigouroz 🔥 It has been a pleasure to work with you 💪

**Pull request merged**